source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionSetRenderer.cs

```diff
@@ -138,10 +145,56 @@ public static void AddActions(Grid uiContainer, IList<AdaptiveAction> actions, A
             }
         }
 
+        public static void AddActionsToRoot(AdaptiveRenderContext context)
```

Comment:
>AddActionsToRoot [](start = 27, length = 16)

this function should be renamed as it is adding the showcards or hiddencards, not the actual actions